### DISCUSSION TITLE
Raw photo not downloaded

### DISCUSSION THREAD

**Discussion by marcolam** [photoprism/photoprism#2982] [Q&A] [answered]
Uploaded one nikon raw photo to https://demo.photoprism.app/ today,it generate thumb files sucessfully but when click download icon nothing happened. thought click edit -files-download just work fine.  jpg files don't have this problem.

the photo titled "Nikon Raw / 2022" with mutiple pingpong tables.

Same on my local setup,raw files won't download, jpg files work fine.tried on several PCs with newest chrome and edge browser.

**Comment by graciousgrey** [comment, marked as answer]
Downloading the RAW from the edit dialog works (https://docs.photoprism.app/user-guide/organize/download/#__tabbed_1_3).

If you want RAWs to be downloaded when using the context menu you need to adapt the download settings in your settings.yml. It is located in your config path.  

The default download settings for downloads from the context menu are:
Originals: true
MediaRaw: false
MediaSidecar: false

If you set MediaRaw to true, RAW files will be downloaded when using the context menu download.
